deployer/debug.py

```diff
@@ -259,5 +260,167 @@ def start_docker_proxy(
+    cluster_name: str = typer.Argument(..., help="Name of cluster to operate on"),
+    hub_name: str = typer.Argument(..., help="Name of hub to operate on"),
+    source_user: str = typer.Option(
+        prompt=True,
+        confirmation_prompt=True,
+        help="Name of source user who's home dir files we are transferring",
+    ),
+    destination_user: str = typer.Option(
+        prompt=True,
+        confirmation_prompt=True,
+        help="Name of destination user in who's home dir we will be moving the files",
+    ),
+):
+    """
+    Copy all the files from a user's old home dir to the new home directory
+    """
+    # Name the pod so we know what to delete when the transfer is done
+    pod_name = f"{cluster_name}-{hub_name}-transfer-shell"
+    pod = {
+        "apiVersion": "v1",
+        "kind": "Pod",
+        "spec": {
+            "terminationGracePeriodSeconds": 1,
+            "automountServiceAccountToken": False,
+            # Make sure the pod and containers can only be ran as the jupyter user
+            # Otherwise the files we copy might end up be owned by another user
+            "securityContext": {"runAsUser": 1000, "runAsGroup": 1000},
+            "volumes": [
+                # This PVC is created by basehub
+                {"name": "home", "persistentVolumeClaim": {"claimName": "home-nfs"}}
+            ],
+            "containers": [
+                {
+                    "name": pod_name,
+                    # Use ubuntu image so we get better GNU cp
+                    "image": "ubuntu:jammy",
+                    "stdin": True,
+                    "stdinOnce": True,
+                    "tty": True,
+                    "volumeMounts": [
+                        {
+                            "name": "home",
+                            "mountPath": "/home",
+                        }
+                    ],
+                    "securityContext": {"allowPrivilegeEscalation": False},
+                }
+            ],
+        },
+    }
+
+    # Command that creates the pod described above
+    create_pod_cmd = [
+        "kubectl",
+        "-n",
+        hub_name,
+        "run",
+        pod_name,
+        "--overrides",
+        json.dumps(pod),
+        "--image",
+        # Use ubuntu image so we get GNU cp and other tools
+        # Should match what we have in our pod definition
+        "ubuntu:jammy",
+        pod_name,
+    ]
+
+    # Command that waits for the pod above to be ready
+    wait_for_pod_cmd = [
+        "kubectl",
+        "-n",
+        hub_name,
+        "wait",
+        "pods",
+        "-l",
+        f"run={pod_name}",
+        "--for",
+        "condition=Ready",
+        "--timeout=90s",
+    ]
+
+    # Command that lists the contents of the source and destination home dirs
+    ls_dirs_cmd = [
+        "kubectl",
+        "-n",
+        hub_name,
+        "exec",
+        pod_name,
+        "--",
+        "ls",
+        "-lat",
```

Comment:
Do we prefer seeing the exact bytes, or do we prefer improved readability?

With `-h` (`--human-readable`), the difference is like below, where its quite impotant for larger files where its hard to know if something is  100GB, or 10GB, 1GB or similar because of the large amount of numbers. The downside is that you can't know if its changed with a single byte, but we have the timestamp to indicate something about this.

I think I suggest we go for `-h`.

```
# with -h
-rw-rw-r--  1 erik erik 1,9K sep 25 16:56 .pre-commit-config.yaml

# without -h
-rw-rw-r--  1 erik erik 1865 sep 25 16:56 .pre-commit-config.yaml
```

```suggestion
        "-lath",
```

Reply:
Agree! Thank you @consideRatio